app/views/users/index.html.erb

```diff
@@ -1 +1,17 @@
-<h1>List of users is here</h1>
\ No newline at end of file
+<h1>List of users is here</h1>
+
+
+# <section class="users">
+#   <%= render @users %>
+# </section> 
```

Comment:
- [ ] I think the following code is not playing any role in your view template file, therefore I would suggest that you remove them to keep your view template clean 😃